code_generator/End2End/schema/exhaustive_test.schema

```diff
@@ -0,0 +1,130 @@
+package improbable.test_schema;
+
+type SomeType {}
+
+// Non-Blittable Components.
+
+component ExhaustiveSingular {
+  id = 197715;
+  bool field1 = 1;
+  float field2 = 2;
+  // bytes field3 = 3; Uncomment when bytes are supported.
```

Comment:
Is this tracked in a ticket anywhere? This and the items below should have JIRA references (and mentions of the need to uncomment the lines in the tickets)

Reply:
I thought we decided to not have ticket references and TODOs scattered through code. @samiwh ? 

The tickets do exist and part of the acceptance criteria is to uncomment the associated lines

Reply:
Then how do we realistically find TODOs in sweeps? :/

Reply:
The reasoning was that code changes frequently enough that often the TODO is no longer relevant or points to the same section of code or gets orphaned.

Personally, I have no problem with TODOs, but we did purge the repo of them and any ticket references

Reply:
I guess the idea is that the ticket should contain enough context that the TODO is unnecessary and the tickets are authoritative (:trollface:) 

We can review the workflow if you strongly oppose however

Reply:
Not having TODO means you have to know about the tickets somehow...the information is stored away from the site of the work.

Reply:
The issue with TODOs in the code is that they might not be cleaned up correctly. All our tickets should have the relevant context to navigate through the code and they should be prioritized accordingly to what we need to do. Having TODOs in the code does provide some value in  terms of knowing which parts need to be rewritten just by looking at the code, but we also risk polluting the code with TODOs that have actually already been fixed (and it makes it easy for us to not include any information directly in the JIRA ticket, but just point to the code)

Reply:
Stale TODOs are much much easier to find than stale generic comments, however.

Reply:
I've gone with `TODO: UTY-123 Short description` format. We should write this down somewhere?